change/@ni-nimble-components-56282a18-4bed-42a1-b324-fee705e7e808.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I did a search for "docs/" and found these paths can also be updated:
- `packages/nimble-components/.prettierignore` has some file patterns
- `packages/nimble-components/CONTRIBUTING.md` links to ./docs/component-status.stories.ts